Libplanet/Blocks/Block.cs

```diff
@@ -472,7 +492,10 @@ internal BlockHeader GetBlockHeader()
                 totalDifficulty: TotalDifficulty,
                 previousHash: previousHashAsArray,
                 txHash: TxHash?.ToByteArray().ToImmutableArray() ?? ImmutableArray<byte>.Empty,
-                hash: Hash.ToByteArray().ToImmutableArray()
+                hash: Hash.ToByteArray().ToImmutableArray(),
+#pragma warning disable MEN002 // Line is too long
+                actionHash: ActionHash?.ToByteArray().ToImmutableArray() ?? ImmutableArray<byte>.Empty
+#pragma warning restore MEN002 // Line is too long
```

Comment:
Rather than suppressing the `MEN002` warning, declaring a new variable would be better, for the meaning of such “long lines are bad” rules.

Reply:
I rename it c7e5fe636f72cf626b280082190f966f09e552b9